app/src/services/market_maker_factory.ts

```diff
@@ -51,13 +60,14 @@ class MarketMakerFactoryService {
     return marketMakerAddress
   }
 
-  getMarkets = async (provider: any): Promise<Market[]> => {
+  getMarkets = async ({ from, to }: GetMarketsOptions): Promise<Market[]> => {
+    console.log(`fetching from ${from} to ${to}`)
```

Comment:
Should be `logger.log`

Reply:
Done.

Reply:
Also added the `no-console` rule to our eslint config btw.